cmd/export/cluster.go

```diff
@@ -106,6 +107,16 @@ func (c *ClusterScopedRbacHandler) prepareForFiltering() {
 			}
 		}
 		c.filteredClusterRoleBindings.objects = &filteredClusterRoleBindings
+	} else {
```

Comment:
I would remove the else block and and a return statement below 109

Otherwise, this makes sense to me

Reply:
fixed in next commit